tailwind.config.js

```diff
@@ -1,9 +1,26 @@
 module.exports = {
-  content: [
-    "./src/**/*.{js,jsx,ts,tsx}",
-  ],
+  content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
+    screens: {
```

Comment:
@nirmalkc to review these settings as they impact things generically.

Reply:
@TejasShekar , is this a standard breakpoint configuration?

Reply:
Apart for `xs` class, the rest is the default breakpoint configuration in tailwindCSS.
Reference : https://tailwindcss.com/docs/screens#:~:text=The%20default%20breakpoints%20are%20inspired%20by%20common%20device%20resolutions%3A

Reply:
@koustov @atapas If all good regarding tailwind standard breakpoints, can we resolve this conversation ?